code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -382,3 +382,236 @@
 	push_data()
 	activate_pin(1)
 	return TRUE
+
+/obj/item/integrated_circuit/input/atmo_scanner
+	name = "integrated atmospheric analyser"
+	desc = "The same atmospheric analysis module that is integrated into every PDA.  \
+	This allows the machine to know the composition, temperature and pressure of the surrounding atmosphere."
+	icon_state = "medscan_adv"
+	complexity = 9
+	inputs = list()
+	outputs = list(
+		"pressure"       = IC_PINTYPE_NUMBER,
+		"temperature" = IC_PINTYPE_NUMBER,
+		"oxygen"         = IC_PINTYPE_NUMBER,
+		"nitrogen"          = IC_PINTYPE_NUMBER,
+		"carbon dioxide"           = IC_PINTYPE_NUMBER,
+		"phoron"           = IC_PINTYPE_NUMBER,
+		"other"           = IC_PINTYPE_NUMBER,
```

Comment:
Trailing comma, this is akin to:
```dm
outputs = list(
		"pressure"       = IC_PINTYPE_NUMBER,
		"temperature" = IC_PINTYPE_NUMBER,
		"oxygen"         = IC_PINTYPE_NUMBER,
		"nitrogen"          = IC_PINTYPE_NUMBER,
		"carbon dioxide"           = IC_PINTYPE_NUMBER,
		"phoron"           = IC_PINTYPE_NUMBER,
		"other"           = IC_PINTYPE_NUMBER,
		null
	)
```

So just remove the comma from the very last member.